measuring/metrics/usage-count.md

```diff
@@ -0,0 +1,24 @@
+# Usage count
```

Comment:
May make it easier to understand what you're looking at when navigating the graph.

```suggestion
# **Metric:** Usage count
```